Source: written by wiget (http://www.mail-archive.com/pld-cvs-commit@lists.pld-linux.org/msg199431.html)
Upstream: No
Reason: Remove pango gir files as they are now installed by x11-libs/pango
--- gir-repository-0.6.5.orig/gir/Makefile.am  2009-10-24 20:59:35.000000000 +0200
+++ gir-repository-0.6.5/gir/Makefile.am       2009-10-24 21:02:41.000000000 +0200
@@ -180,7 +180,7 @@
 #poppler
 if BUILD_POPPLER
 POPPLER_INCLUDEDIR=`pkg-config --variable=includedir poppler-glib`/poppler
-Poppler-0.8.gir: Gdk-2.0.gir Pango-1.0.gir $(INTROSPECTION_SCANNER)
+Poppler-0.8.gir: Gdk-2.0.gir $(INTROSPECTION_SCANNER)
 	$(INTROSPECTION_SCANNER) -v --namespace Poppler --nsversion=0.8 \
 	    --add-include-path=$(srcdir) --add-include-path=. \
              --include=GObject-2.0 \
@@ -266,7 +266,7 @@
 
 lib_LTLIBRARIES += libgirepo-Gdk-custom.la
 GDK_INCLUDEDIR=`pkg-config --variable=includedir gdk-$(GDK_TARGET)-2.0`/gtk-2.0
-Gdk-2.0.gir: Pango-1.0.gir GdkPixbuf-2.0.gir libgirepo-Gdk-custom.la $(INTROSPECTION_SCANNER)
+Gdk-2.0.gir: GdkPixbuf-2.0.gir libgirepo-Gdk-custom.la $(INTROSPECTION_SCANNER)
 	$(INTROSPECTION_SCANNER) -v --namespace Gdk --nsversion=2.0 \
 	    --add-include-path=$(srcdir) --add-include-path=. \
              --include=Gio-2.0 \
@@ -291,7 +291,7 @@
 
 lib_LTLIBRARIES += libgirepo-Gtk-custom.la
 GTK_INCLUDEDIR=`pkg-config --variable=includedir gtk+-$(GDK_TARGET)-2.0`/gtk-2.0
-Gtk-2.0.gir: Pango-1.0.gir Atk-1.0.gir GdkPixbuf-2.0.gir Gdk-2.0.gir libgirepo-Gtk-custom.la $(INTROSPECTION_SCANNER)
+Gtk-2.0.gir: Atk-1.0.gir GdkPixbuf-2.0.gir Gdk-2.0.gir libgirepo-Gtk-custom.la $(INTROSPECTION_SCANNER)
 	$(INTROSPECTION_SCANNER) -v --namespace Gtk --nsversion=2.0 \
 	    --add-include-path=$(srcdir) --add-include-path=. \
              --include=Gio-2.0 \
@@ -429,7 +429,7 @@
              --output $@ \
              $(CLUTTER_INCLUDEDIR)/clutter-0.8/clutter/json/*.h
 
-Clutter-0.8.gir: libgirepo-Clutter-custom.la ClutterJson-0.8.gir GdkPixbuf-2.0.gir Pango-1.0.gir PangoCairo-1.0.gir $(INTROSPECTION_SCANNER) clutter-0.8.c
+Clutter-0.8.gir: libgirepo-Clutter-custom.la ClutterJson-0.8.gir GdkPixbuf-2.0.gir $(INTROSPECTION_SCANNER) clutter-0.8.c
 	$(INTROSPECTION_SCANNER) -v --namespace Clutter --nsversion=0.8 \
 	    --add-include-path=$(srcdir) --add-include-path=. \
              --include=GL-1.0 \

